tests/lit.cfg

```diff
@@ -70,7 +70,6 @@ def inferClang(PATH):
 cxx_options = ' ' + ' '.join([
   "-I%s" % config.gtest_src_dir,
   "-DGTEST_HAS_TR1_TUPLE=0",
-  "-stdlib=libc++",
```

Comment:
yes, this branch has been tested on Ubuntu 14.04 with USE_LIBCXX=ON.   -stdlib=libc++ is provided by the hcc-config tool